docs/learn/avalanche/avalanche-consensus.md

```diff
@@ -147,120 +175,24 @@ number of participants in the network, the number of consensus messages sent rem
-
-![Working example 6](/img/cons-07-Consensus_Doc_txY-1.png)
-
-Now the node learns about a new transaction, **Z**, and it does the same thing as before. It queries
-_k_ nodes, gets back an α majority response, and updates the DAG.
+Everything discussed to this point is how Avalanche is described in [the Avalanche
+white-paper](https://assets-global.website-files.com/5d80307810123f5ffbb34d6e/6009805681b416f34dcae012_Avalanche%20Consensus%20Whitepaper.pdf).
+The implementation of the Avalanche consensus protocol by Ava Labs (namely in AvalancheGo) has some
+optimizations for latency and throughput.
 
-![Working example 7](/img/cons-08-Consensus_Doc_txY-5.png)
+### Blocks
 
-Transaction **Z** gets a chit. It also has a confidence of `1` and `1` consecutive success. The
-processing ancestors are updated, too. No transactions have `4` consecutive successes so no
-ancestors are accepted.
+A block is a fundamental component that forms the structure of a blockchain. It serves as a container
+or data structure that holds a collection of transactions or other relevant information. Each block
+is cryptographically linked to the previous block, creating a chain of blocks, hence the term "blockchain."
 
-### Vertices
+In addition to storing a reference of its parent, a block contains a set of transactions. These
+transactions can represent various types of information, such as financial transactions, smart 
+contract operations, or data storage requests.  
 
-Everything discussed to this point is how Avalanche is described in [the Avalanche
-white-paper](https://assets-global.website-files.com/5d80307810123f5ffbb34d6e/6009805681b416f34dcae012_Avalanche%20Consensus%20Whitepaper.pdf).
-The implementation of the Avalanche consensus protocol by Ava Labs (namely in AvalancheGo) has some
-optimizations for latency and throughput. The most important optimization is the use of
-**vertices**. A vertex is like a block in a linear blockchain. It contains the hashes of its
-parents, and it contains a list of transactions. Vertices allow transactions to be batched and voted
-on in groups rather than one by one. The DAG is composed of vertices, and the protocol works very
-similar to how it's described above.
-
-If a node receives a vote for a vertex, it counts as a vote for all the transactions in a vertex,
-and votes are applied transitively upward. A vertex is accepted when all the transactions which are
-in it are accepted. If a vertex contains a rejected transaction then it is rejected and all of its
-descendants are rejected. If a vertex is rejected, any valid transactions are re-issued into a new
-vertex which is not the child of a rejected vertex. New vertices are appended to preferred vertices.
+If a node receives a vote for a block, it counts as a vote for all the transactions in a block.
+A block is accepted when all the transactions which are in it are accepted. If a block is rejected,
+any valid transactions are re-issued into a new block.
```

Comment:
It's not actually strictly required that the transactions in a rejected block are put into a new block and reissued, though in practice we do that with all of our VMs. It's not part of Snowman consensus, it's a VM level decision.

```suggestion
If a node receives a vote for a block, it also counts as a vote for all of the block's ancestors (its parent, the parents' parent, etc.).
```